src/app/about/page.tsx

```diff
@@ -26,11 +27,14 @@ import { focusAreasData } from './data/focusAreasData'
 import { reportsData } from './data/reportsData'
 import { generateStructuredData } from './utils/generateStructuredData'
 
-const { header, seo } = attributes
+const aboutPageData = getPageMarkdownData({
+  slug: 'about',
+  zodParser: GenericPageDataSchema.parse,
+})
```

Comment:
Good thinking.

`PATHS.ABOUT.path` gives us `/about`. We want `about`, which is the file's name. I'm not sure we can rely entirely on `PATHS` here because the URL paths don't match the file paths 1:1:
- `home.md` is `/`
- `maturity-model.ms` is `/security/maturity-model`

Reply:
I do think @barbaraperic has a point here. I'm still figuring out this PR, but why don't we have something like:


```
function getSlugFromPath(path: PathValues) {
  const pathConfig = Object.values(PATHS).find((config) => config.path === path)
  if (!pathConfig) {
    throw new Error(`Path config not found for path: ${path}`)
  }

  return (pathConfig.mainContentPath ?? path).replace(
    `${CONTENT_PAGES_ROOT}`,
    '',
  )
}

export function getPageMarkdownData<T>({ slug, zodParser }: GetPageData<T>) {
  const mainContentPath = getSlugFromPath(slug as PathValues)
  return getMarkdownData({
    slug: mainContentPath,
    directoryPath: CONTENT_PAGES_ROOT,
    zodParser,
  })
}
```

This was we can do:

```
const aboutPageData = getPageMarkdownData({
  slug: PATHS.ABOUT.path,
  zodParser: GenericPageDataSchema.parse,
})
```

Reply:
✅

Reply:
`getPageMarkdownData` now accepts `PATHS.ABOUT` + a Zod parser function.